tests/e2e/test_rosacli_account_roles.go

```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
+				"--version", "4.8.10",
+				"-y")
+			Expect(err).ToNot(BeNil())
+			Expect(output.String()).To(ContainSubstring("A valid policy version number must be specified"))
+
+			By("Create account roles with invalid channel group")
+			versionService := rosaClient.Version
+			// get stable channel version
+			versionList, err := versionService.ListAndReflectVersions(rosacli.VersionChannelGroupStable, true)
+			Expect(err).To(BeNil())
+			defaultVersion := versionList.DefaultVersion()
+			Expect(defaultVersion).ToNot(BeNil())
+
+			output, err = ocmResourceService.CreateAccountRole("--mode", "auto",
+				"--prefix", "ocp75246",
+				"--version", defaultVersion.Version,
+				"--channel-group", "fakecg",
+				"-y")
+			Expect(err).ToNot(BeNil())
+			Expect(output.String()).To(ContainSubstring("Error getting version: could not find versions for the " +
+				"provided channel-group: 'fakecg'"))
+
+		})
+
+	It("to create/Upgrade account-roles by setting version and channel-group via rosacli - [id:54469]",
+		labels.High, labels.Runtime.OCMResources,
+		func() {
+			defer func() {
+				By("Cleanup created account-roles in the test case")
+				if len(accountRolePrefixesNeedCleanup) > 0 {
+					for _, v := range accountRolePrefixesNeedCleanup {
+						_, err := ocmResourceService.DeleteAccountRole("--mode", "auto",
+							"--prefix", v,
+							"-y")
+
+						Expect(err).To(BeNil())
+					}
+				}
+			}()
+
+			By("Prepare y-1 version for testing")
+			versionService := rosaClient.Version
+			// get stable channel version
+			versionListS, err := versionService.ListAndReflectVersions(rosacli.VersionChannelGroupStable, true)
+			Expect(err).To(BeNil())
+			defaultVersionS := versionListS.DefaultVersion()
+			Expect(defaultVersionS).ToNot(BeNil())
+			_, _, upgradeVersionS, err := defaultVersionS.MajorMinor()
+			Expect(err).To(BeNil())
+			versionS, err := versionListS.FindNearestBackwardMinorVersion(defaultVersionS.Version, 1, true)
+			Expect(err).To(BeNil())
+			Expect(versionS).NotTo(BeNil())
+			_, _, versionStable, err := versionS.MajorMinor()
+			Expect(err).To(BeNil())
+
+			// get candidate channel version
+			versionListC, err := versionService.ListAndReflectVersions(rosacli.VersionChannelGroupCandidate, true)
+			Expect(err).To(BeNil())
+			defaultVersionC := versionListC.DefaultVersion()
+			Expect(defaultVersionC).ToNot(BeNil())
+			_, _, upgradeVersionC, err := defaultVersionC.MajorMinor()
+			Expect(err).To(BeNil())
+			versionC, err := versionListS.FindNearestBackwardMinorVersion(defaultVersionC.Version, 1, true)
+			Expect(err).To(BeNil())
+			Expect(versionC).NotTo(BeNil())
+			_, _, versionCandidate, err := versionC.MajorMinor()
+			Expect(err).To(BeNil())
+
+			channelVersion := map[string]string{"stable": versionStable, "candidate": versionCandidate}
+
+			By("Create/upgrade account role with version and channel group in auto mode")
+			for c, v := range channelVersion {
+				By("Create account roles in auto mode")
```

Comment:
Can you add the channel group to the `By` titles ? That way, it is easier to follow

Reply:
add channel group to By titles.